### ISSUE TITLE
Issue using loss backend with custom logic

### ISSUE THREAD

**Issue by jh4n23** [vehicle-lang/vehicle#1226]
Disclaimer: this is almost certainly down to my lack of understanding of how to implement this.

I am trying to use QLL as a custom logic (I don't believe it is supported in the current release). It is defined in my Vehicle specification like so:

[mnist-robustness.txt](https://github.com/user-attachments/files/31185292/mnist-robustness.txt)

In [this python file](https://github.com/user-attachments/files/31185361/pdt_classifier.py), I attempt to load the specification and generate the loss function like so:

```
def get_spec():
    return loss_pt.load_specification(
        SPEC_PATH,
        logic=vcl.CustomDifferentiableLogic("qllAdditive")
    )
```

```
constraint_loss = torch.stack(constraint_loss_fn(
    n=images.shape[0],
    classifier=self.network,
    epsilon=torch.tensor(0.1 / 0.3081),
    trainingImages=images.squeeze(1),
    trainingLabels=labels,
    p=torch.tensor(5.0),
    low=torch.tensor(-0.4242),
    high=torch.tensor(2.8215)
)).mean()
```

This causes the following error:

```
full() received an invalid combination of arguments - got (size=int, dtype=torch.dtype, fill_value=float, ), but expected one of:
 * (tuple of ints size, Number fill_value, *, tuple of names names, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
 * (tuple of ints size, Number fill_value, *, Tensor out = None, torch.dtype dtype = None, torch.layout layout = None, torch.device device = None, bool pin_memory = False, bool requires_grad = False)
```

**Comment by MatthewDaggitt** [comment]
Is this error being thrown during the call to `constraint_loss_fn` or is it being thrown by `mean`/`stack` on the outside? If the latter, can you first print the result of calling `constraint_loss_fn` and post the output here?
